zaldis/MAcademy2021
# Add Reader & Book models

- Add Reader model
- Add Book model

**Add endpoints:**
- _`/`_
root page
- _`/books`_
list of available books
- _`/readers`_
list of authenticated readers
- _`/users`_
list of users with permissions

## HOW TO TEST
1. Run migrate command: `python manage.py migrate`.
2. Run Django server: `python manage.py runserver`.
3. Fill db with necessary data.
4. Check that all endpoints are displayed properly.
